runtime/flang/CMakeLists.txt

```diff
@@ -475,15 +475,23 @@ SET(FTN_SUPPORT
   xfer.c
   init.c
   xfer_rpm1.c
+  )
+
+add_library(iso_c_bind OBJECT
+  iso_c_bind.F95
+)
+
+add_library(ieee_arithmetic OBJECT
   ieee_arithmetic.F95
   ieee_exceptions.F95
-  iso_c_bind.F95
-  )
+)
 
 add_flang_library(flang_static
   ${FTN_INTRINSICS}
   ${FTN_SUPPORT}
   ${SHARED_SOURCES}
+  $<TARGET_OBJECTS:iso_c_bind>
```

Comment:
This is wrong. These objects are built with different compile options than the library.